src/convertToObject.js

```diff
@@ -6,7 +6,20 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const styles = sourceString
+    .split(';')
+    .map((style) => style.split(':'))
+    .reduce((acc, [key, value]) => {
+      if ((key, value)) {
+        const [_key, _value] = [key.trim(), value.trim()];
```

Comment:
1) why do u need these underlines?
is this some special code style?
share a link to it, and explain

2) why can't we immediately use it in our condition on the line?
do u need to take it as a separate term?